tests/test_task_worker.py

```diff
@@ -11,10 +11,13 @@
 from prefect.exceptions import MissingResult
 from prefect.filesystems import LocalFileSystem
 from prefect.futures import PrefectDistributedFuture
+from prefect.settings import PREFECT_API_URL, temporary_settings
 from prefect.states import Running
 from prefect.task_worker import TaskWorker, serve
 from prefect.tasks import task_input_hash
 
+pytestmark = pytest.mark.usefixtures("use_hosted_api_server")
```

Comment:
i think we should always use a hosted server for these tests since `task_workers` should not work against the ephemeral api?